.github/workflows/pull-request.yml

```diff
@@ -9,10 +9,13 @@ on:
 # This doesn't work in Actions, however, so we set this env var to force the Headless runner
 env:
   TEST_BROWSERS: ChromeHeadless
-  
+
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
```

Comment:
will this matrix cover all possibilities?

Reply:
What possibilities am I missing?

Reply:
I am just not sure whether `-lastest` is able to cover all OS possibilities. Or we may don't have to cover all?